### PR TITLE
Corrected class schedule

### PR DESCRIPTION
**Pull request by NatalyaTretyak**
Changed class schedule

### CODE REVIEW

File: src/app/admin-panel/admin-panel/attach-teacher/attach-teacher.component.ts [hunk 1]
```diff
@@ -48,7 +50,10 @@ export class AttachTeacherComponent implements OnInit {
     this.filteredSubjects = [];
     for (let i = 0; i < this.subjects.length; i++) {
       const subject = this.subjects[i];
-      if (subject.subjectName.toLowerCase().indexOf(event.query.toLowerCase()) === 0) {
+      if (
+        subject.subjectName.toLowerCase().indexOf(event.query.toLowerCase()) ===
```
Comment (dmitryy90) [inline src/app/admin-panel/admin-panel/attach-teacher/attach-teacher.component.ts]: format file

File: src/app/admin-panel/admin-panel/class-schedule/class-schedule.component.html [hunk 2]
```diff
@@ -1,47 +1,263 @@
...
+        >
+          <i class="far fa-trash-alt"></i>
+        </button>
+      </p>
     </p-fieldset>
-    <p-fieldset legend="Вівторок">
-      <app-day [subjectsList]="weekDays.tuesdaySubjects"></app-day>
+
+    <p-fieldset class="form-control day" legend="Вівторок">
+      <p
```
Comment (dmitryy90) [inline src/app/admin-panel/admin-panel/class-schedule/class-schedule.component.html]: use semantic html

File: src/app/admin-panel/admin-panel/class-schedule/class-schedule.component.html [hunk 3]
```diff
@@ -1,47 +1,263 @@
...
+          pButton
+          type="button"
+          (click)="delSubject($event, schedule.mondaySubjects, i)"
+        >
+          <i class="far fa-trash-alt"></i>
+        </button>
+      </p>
     </p-fieldset>
-    <p-fieldset legend="Вівторок">
-      <app-day [subjectsList]="weekDays.tuesdaySubjects"></app-day>
```
Comment (dmitryy90) [inline src/app/admin-panel/admin-panel/class-schedule/class-schedule.component.html]: why did you remove app-day component?

File: src/app/admin-panel/admin-panel/class-schedule/class-schedule.component.ts [hunk 4]
```diff
@@ -1,52 +1,118 @@
...
+    }
+    return [year, month, day].join('-');
+  }
+
+  // request to add a list of subjects
+  getScheduleSubjects(): void {
+    this.scheduleService.getScheduleSubjects().subscribe(data => {
+      this.subjects = data;
+      // add an empty field
+      // this.subjects = [new Subject('', '-'), ...data];
```
Comment (dmitryy90) [inline src/app/admin-panel/admin-panel/class-schedule/class-schedule.component.ts]: commented code

Comment (NatalyaTretyak) [reply src/app/admin-panel/admin-panel/class-schedule/class-schedule.component.ts]: Deleted commented code

File: src/app/admin-panel/admin-panel/class-schedule/class-schedule.component.html (hunk 2 above)
Comment (NatalyaTretyak) [reply src/app/admin-panel/admin-panel/class-schedule/class-schedule.component.html]: Fixed html

File: src/app/admin-panel/admin-panel/class-schedule/class-schedule.component.html (hunk 3 above)
Comment (NatalyaTretyak) [reply src/app/admin-panel/admin-panel/class-schedule/class-schedule.component.html]: I will return it

File: src/app/admin-panel/admin-panel/attach-teacher/attach-teacher.component.ts (hunk 1 above)
Comment (NatalyaTretyak) [reply src/app/admin-panel/admin-panel/attach-teacher/attach-teacher.component.ts]: Fixed format
